### PR TITLE
Circle ci integration

### PR DESCRIPTION
**Pull request by Namiii**
**TASK**=> T13623

**SUMMARY**
Added CircleCi configuration 

👾**Note** 👾
We are still not getting a pass from CircleCi but that is because there are failing tests and not because there is no configuration.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            - ~/.gradle
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+      - run:
+          name: Run Tests
+          command: ./gradlew lint test
+      - store_artifacts:
+          path: app/build/reports
+          destination: reports
+      - store_test_results:
+          path: app/build/test-results
```
Comment (nuxzero) [inline .circleci/config.yml]: Could you use SDK path instead sample app path?

Comment (Namiii) [reply .circleci/config.yml]: As in `~/` ?

Comment (nuxzero) [reply .circleci/config.yml]: I think that represent to root path. exactly I asking for use `sdk` instead of `app`.

Comment (nuxzero) [reply .circleci/config.yml]: Or maybe it can add multiple path.

Comment (Namiii) [reply .circleci/config.yml]: Ahhhh now I understand what you mean 🤦‍♂ I'll change it

**Pull request merged**